### ISSUE TITLE
No module named ‘lghorizon’

### ISSUE THREAD

**Issue by grewhit25** [Sholofly/lghorizon#160]
**Describe the bug**
I have followed the installation instructions and have installed the latest version using HACS.

However, when I try to add ‘LG Horizon’ home-assistant integration.
The following error is returned by the frontend:
```logs
Error
Config flow could not be loaded: {"message":"Invalid handler specified"}”
```
HA error log output:
```logs
2024-11-14 14:30:05.251 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading flow for integration lghorizon: No module named 'lghorizon'
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version of the component**
The version of the component as it is installed on your HA

v0.7.4

HA container version
2024.11.1

**Logging**
Enable debug logging and add your log. DO NOT SHARE SENSITIVE DATA SUCH AS PASSWORDS AND ACCOUNT INFO!
You can enable logging by setting up the logging component in HA as follows:
```yaml
logger:
  default: warning
  logs:
    lghorizon: debug
```

**Additional context**

**Comment by nrosier** [comment]
Same issue. Anyone know how to fix?

**Comment by Sholofly** [comment]
Sorry for this sys admin question, but did you try to uninstall and install the component again?

**Comment by grewhit25** [comment]
Yes I have removed and re downloaded the component. The issue remains. 
Any idea what the problem might be?

**Comment by grewhit25** [comment]
Got this working with the following workaround.

I exec’d into the home-assistant container, then ran:
``` code
pip install lghorizon
```

I had a similar problem with a different component a little while back where I applied the above procedure; that issue was resolved in a later home-assistant build.

**Comment by grewhit25** [comment]
This issue appears to be fixed by the latest HA build 2025.1.0.

Closing.
